### PR TITLE
[doc] Update FPGA setup docs.

### PR DESCRIPTION
**Pull request by jadephilipoom**
Updates after some issues setting up a new FPGA:
- Add a note to the CW310 setup stating that you should run `set-pll`
- Add `test_rom` suffix to test targets
- Include some troubleshooting notes that suggest adding the `--rcfile=` option and pressing `USR_RST`

### CODE REVIEW

File: doc/getting_started/setup_fpga.md [hunk 1]
```diff
@@ -136,6 +136,25 @@ When properly connected, `dmesg` should identify the board, not show any errors,
...
+bazel test --test_output=streamed //sw/device/tests:uart_smoketest_fpga_cw310_test_rom
+```
+
+### Troubleshooting
+
+If the tests/demo aren't working on the FPGA (especially if you get an error like `SFDP header contains incorrect signature`) then try adding `--rcfile=` to the `set-pll` command:
+```console
+bazel run //sw/host/opentitantool -- --rcfile= --interface=cw310 fpga set-pll
+```
+It's also worth pressing the `USR_RST` button on the FPGA if you face continued errors.
```
Comment (drewmacrae) [inline doc/getting_started/setup_fpga.md:156]: Do you mean USR_RESET or USB_RST? I've found USB_RST useful in the past when setting up the board and meant to mention it in the documentation somewhere when I set up my FPGA.

**Review by drewmacrae — APPROVED** [review]

**Review by dmcardle — APPROVED** [review]
LGTM! Thanks for improving the docs :)

File: doc/getting_started/setup_fpga.md (hunk 1 above)
Comment (jadephilipoom) [reply doc/getting_started/setup_fpga.md:156]: I'll just add both here. `USR_RST` is the one I ended up pressing, but no idea if it did anything and I've also found `USB_RST` helpful in the past.

Comment (drewmacrae) [reply doc/getting_started/setup_fpga.md]: Just a further nit here: Silkscreen labels can be hard to correctly read and the USB_RST label is fairly small, On my CW310, the other label reads "USR_RESET" and being careful about the labels gives people a better chance of reading it correctly.

Comment (drewmacrae) [reply doc/getting_started/setup_fpga.md]: I can confirm: "USB_RST" is mentioned elsewhere

Comment (drewmacrae) [reply doc/getting_started/setup_fpga.md]: Where it's associated with "SW5"

**Comment by ballifatih** [comment]
I might be stating the obvious here but I suppose the previous guide assumed that one has `--interface=cw310` in the config file (which I think is the reason I didn't have to pass it myself when I ran `bazel run //sw/host/opentitantool fpga set-pll`).

https://github.com/lowRISC/opentitan/blob/b15fa805be3bae4528310e614372d91761faef28/doc/getting_started/setup_fpga.md?plain=1#L195-L199

I am not really familiar with the issue and I was just following the suggestion by @mundaym for fixing my local issue.

**Review by ballifatih — APPROVED** [review]

**Pull request merged**